pkg/githubtemplate/github_template.go

```diff
@@ -10,8 +10,8 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-// Find returns the list of template file paths
-func Find(rootDir string, name string) []string {
+// FindNonLegacy returns the list of template file paths from the template folder (according to the "upgraded multiple template builder")
```

Comment:
GitHub refers to the original way of `PULL_REQUEST_TEMPLATE.md` and `ISSUE_TEMPLATE.md` files as 'legacy' and the new method of `PULL_REQUEST_TEMPLATE` and `ISSUE_TEMPLATE` folders containing the template files as "upgraded multiple template builder".  
I have hence called them as legacy and nonlegacy